### PR TITLE
add infrastructure-components make fmt

### PR DESCRIPTION
**Pull request by sadysnaat**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/cluster-api-provider-maas/pull/7#" title="Author self-approved">sadysnaat</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **jzhoucliqr**
You can assign the PR to them by writing `/assign @jzhoucliqr` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Fcluster-api-provider-maas).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/cluster-api-provider-maas/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @sadysnaat. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Fcluster-api-provider-maas).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by spectro-prow** [comment]
@sadysnaat: Cannot trigger testing until a trusted user reviews the PR and leaves an `/ok-to-test` message.

<details>

In response to [this](https://github.com/spectrocloud/cluster-api-provider-maas/pull/7#issuecomment-890709359):

>/ok-to-test


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: Makefile
Comment (jzhoucliqr) [inline Makefile]: how to set the correct image during a release build?

File: config/default/kustomization.yaml
Comment (jzhoucliqr) [inline config/default/kustomization.yaml]: do you think we should use capm or capmaas ? I don't know personally, just asking.  
metal3 provider is using capm3.

File: Makefile
Comment (sadysnaat) [reply Makefile]: Ack added release manifest make target

File: config/default/kustomization.yaml
Comment (sadysnaat) [reply config/default/kustomization.yaml]: Ack

**Review by jzhoucliqr — APPROVED** [review]
/approve

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/cluster-api-provider-maas/pull/7#pullrequestreview-720789583" title="Approved">jzhoucliqr</a>*, *<a href="https://github.com/spectrocloud/cluster-api-provider-maas/pull/7#" title="Author self-approved">sadysnaat</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Fcluster-api-provider-maas).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/spectrocloud/cluster-api-provider-maas/blob/main/OWNERS)~~ [jzhoucliqr]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
